packages/api/src/external/carequality/ihe-to-external-gw-document-query.ts

```diff
@@ -0,0 +1,6 @@
+import { IHEToExternalGwDocumentQuery as IHEToExternalGwDocumentQueryCore } from "@metriport/core/src/external/carequality/ihe-result";
+import { BaseDomainCreate } from "@metriport/core/domain/base-domain";
+
+export interface IHEToExternalGwDocumentQuery
```

Comment:
ditto naming: recommend `IHEOutboundDocumentQuery` here and throughout with similar name